src/llama.cpp

```diff
@@ -2907,6 +2907,9 @@ struct llama_model {
     // for quantize-stats only
     std::vector<std::pair<std::string, struct ggml_tensor *>> tensors_by_name;
 
+    uint64_t n_bytes = 0;
+    uint64_t n_elements = 0;
```

Comment:
```cpp
struct llama_model_loader {
    int64_t n_elements = 0; 
    size_t  n_bytes    = 0; 
};
```
`llama_model_loader` uses `int64_t` and `size_t` for these values

```cpp
uint64_t llama_model_size(const struct llama_model *model) { } 
uint64_t llama_model_n_params(const struct llama_model *model) { }
```

i think `n_elements` could safely be casted to `uint64_t`, so the api doesn't have to change and I used `uint64_t` for both.

```cpp
size_t llama_model_size(const struct llama_model *model) { } 
int64_t llama_model_n_params(const struct llama_model *model) { }
```

if you want to avoid an implicit cast you will have to change the api